src/Executables/PROMPTS.ps1

```diff
@@ -0,0 +1,93 @@
+$sh = New-Object -ComObject "Wscript.Shell"
+
+<#
+		--------------------------
+				Mitigations
+		--------------------------
+#>
+
+$WindowTitle = 'Security Mitigations Prompt - Atlas'
+
+$Message = @'
+Would you like to disable security CPU mitigations/fixes for vulnerabilities like Meltdown and Spectre?
+
+This is mostly beneficial on older CPUs, recent CPUs have these fixes implemented in hardware. In some cases (like AMD Zen 4 CPUs), it can be significantly worse for performance to disable mitigations.
```

Comment:
Is "it can be significantly worse for performance to **disable** mitigations." not meant to be "it can be significantly worse for performance to **enable** mitigations."?

Reply:
Sort of unrelated to the pull request, but yes, it is meant to be that way. https://www.tomshardware.com/news/ryzen-7000-security-mitigations-faster-on-linux

Reply:
Oh alright...interesting. I thought that enabling them would always worsen performance. 
Thanks for quick clarification.